capstone-frontend/src/components/SearchUserModal.jsx

```diff
@@ -42,12 +41,16 @@ export class SearchUserModal extends Component {
       if (typeof searchResults === 'undefined') {
         searchResults = [];
       }
+      // Owner cannot be a Collaborator
+      else if (this.props.type === 'booklists' && this.props.bookList) {
+        searchResults = searchResults.filter(searchItem => searchItem.id !== window.localStorage.getItem('userID'))
+      }
 
       if (searchResults.length !== 0) {
-        this.setState({ searchResults, fetchingUsers: false, resultsFound: true }) // Guilty
```

Comment:
What do you mean by guilty?

Reply:
https://github.com/STEP-Pod-Capstone-Project/Capstone-Project/pull/107#discussion_r461796228

Reply:
https://github.com/STEP-Pod-Capstone-Project/Capstone-Project/pull/107#discussion_r461807058